trin-history/src/main.rs

```diff
@@ -0,0 +1,6 @@
+use trin_history;
+
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn std::error::Error>> {
+    return trin_history::main().await;
```

Comment:
Nit: Maybe the following is more idiomatic? Similarly for state.

```suggestion
    trin_history::main().await
```

Reply:
thanks for the nit, added a commit to make those changes